src/OXRS_LCD.cpp

```diff
@@ -281,6 +276,21 @@ void OXRS_LCD::update(void)
     }
   }
  
+  // check if ethernet link status has changed
+  uint8_t tmp = Ethernet.linkStatus();
```

Comment:
I was thinking this code should be inside `show_ethernet()` - and you call that inside the calling firmware `loop()`. The reason being we might want to support WiFi connections at some point - so it makes it nice and clean to just have `show_ethernet()` or `show_wifi()` and that gets called every loop.

You can then remove the `screen.show_ethernet()` from the firmware `setup()` routine, since it will be called on the first program loop - i.e. just a single reference to this method in the firmware sketch.

Then it will be nice and easy to add a `screen.show_wifi()` equivalent if someone needs it.

Thoughts?

Reply:
done
added hooks for wifi (nothing working for wifi implemented)